drivers/remote/driver.go

```diff
@@ -146,6 +147,9 @@ func (d *driver) EndpointOperInfo(nid, eid types.UUID) (map[string]interface{},
 	if err := d.call("EndpointOperInfo", info, &res); err != nil {
 		return nil, err
 	}
+	if val, ok := res.Value["MacAddress"]; ok {
+		res.Value[netlabel.MacAddress], _ = net.ParseMAC(val.(string))
+	}
```

Comment:
I think the expectation to use keys as defined in netlabel (when needed) is reasonable. We need to document that as part of documentation for developing network plugins.

What I think may've caused a bit of confusion is how the request and response for creating endpoints are formed and parsed. The keys used in such requests/responses need to be well defined as well and right now they are in the form of Address, MacAddress, etc. So for example the mac address of an interface is expected  by func (r *createEndpointResponse) parseInterfaces() to have been returned in the response of a create endpoint with the MacAddress key as part of the interface info.